### PR TITLE
#12 - License (and contrib) readme

### PR DESCRIPTION
**Pull request by archae0pteryx**
Self explanatory

## PR Comments Emoji Legend

😻 - `:heart_cat_eyes:` Compliment to code/idea/etc
♻️ - `:recycle:` Non-blocking proposed refactor
➕ - `:heavy_plus_sign:` Non-blocking proposed addition
🔥 - `:fire:` Non-blocking proposed removal
❓ - `:question:` Non-blocking question
⚠️ - `:warning:` Blocking comment that must be addressed before PR

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Copyright (YEAR) (COPYRIGHT HOLDER)
```
Comment (craineum) [inline LICENSE.md:1]: ❓ This looks like the MIT license, is that what it is? If so, we should probably state that somewhere?!?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
+# Introduction
+
+## Write something nice here
+
+>First off, thank you for considering contributing to Active Admin. It's people like you that make Active Admin such a great tool.
```
Comment (craineum) [inline CONTRIBUTING.md:5]: ♻️ This needs lots of updating for our app.

File: LICENSE.md (hunk 1 above)
Comment (archae0pteryx) [reply LICENSE.md:1]: It is. Good point. I will put a link to the license templates as well.

File: CONTRIBUTING.md (hunk 2 above)
Comment (archae0pteryx) [reply CONTRIBUTING.md:5]: Wanna add a `.template` extension to it for now?

Comment (craineum) [reply CONTRIBUTING.md:5]: Sure, that works.

**Pull request merged**